### PR TITLE
Removed un-used last_message_ids from queryChannels api call

### PR DESCRIPTION
**Pull request by vishalnarkhede**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

**Review by ferhatelmas — APPROVED** [review]
LGTM

File: src/client.ts [hunk 1]
```diff
@@ -1050,7 +1041,7 @@ export class StreamChat<
       await this.queryChannels(
         { cid: { $in: cids } } as ChannelFilters<ChannelType, UserType, CommandType>,
         { last_message_at: -1 },
-        { limit: 30, recovery: true, last_message_ids: lastMessageIDs },
+        { limit: 30, recovery: true },
```
Comment (ferhatelmas) [inline src/client.ts:1044]: while we're here, is recovery used ?

Comment (vishalnarkhede) [reply src/client.ts:1044]: @thesyncim ? ^^

Comment (thesyncim) [reply src/client.ts]: this field was used in the early development stages (even before we went to production), its safe to remove

**Review by thesyncim — APPROVED** [review]

**Pull request merged**